comet_core/api_v0.py

```
@@ -141,3 +142,42 @@ def dbhealth_check():
         return jsonify({'status': 'error', 'msg': 'dbhealth_check failed'}), 500
 
     return 'Comet-API-v0'
+
+
+@bp.route('/acknowledge', methods=('POST',))
+@requires_auth
+def acknowledge():
+    """Mark the given fingerprint as acknowledge
+
+    Returns:
+        str: the HTTP response string
+    """
+    try:
+        fingerprint = request.get_json()['fingerprint']
+        get_db().ignore_event_fingerprint(fingerprint,
+                                          IgnoreFingerprintRecord.ACKNOWLEDGE)
+    except Exception as _:  # pylint: disable=broad-except
```

Unless you need absolutely need to, swallowing errors is an anti-pattern and should be avoided. This error should be logged internally (but not sent to the user) to aid in debugging why this thing failed.

---

It's a code smell if you often have to disable the same linter rule

---

@gbrindisi it looks like you may have set this precedent and @nuritizra is following it

---

@OmarDarwish where?

---

@nuritizra but @OmarDarwish is right, can you address this as well?

Something like:
```
except Exception as e:
    LOG.debug(....)
```

---

> @OmarDarwish where?

[This](https://github.com/nuritizra/comet-core/commit/7c00be126552fa0fe809c712d132c65cd0ee567b#diff-ef8992e04e31436df508e372773412ecR90) seems to be the first commit for these files. You can see `pylist: disable=broad-except` across the file set. Maybe the blame points at you because you were the first to port this to github?